### DISCUSSION TITLE
where is /api/me

### DISCUSSION THREAD

**Discussion by BonderTam** [logto-io/logto#1924] [Q&A] [answered]
local env:  
windows10,  node app
logto docker updated to tag edge;

when non-admin user signed in, how to get it's  userInfo? for it can not access Logto API

![image](https://user-images.githubusercontent.com/249648/190045567-87f65ffb-0aa1-469d-a888-cb2fe366d065.png)

**Comment by simeng-li** [comment, marked as answer]
try `loclhaost:3001/oidc/me `

**Reply by BonderTam** [reply]
> try `loclhaost:3001/oidc/me `

![image](https://user-images.githubusercontent.com/249648/190075980-a4a964ca-0d7a-45ad-a97a-dafdd1cf72f9.png)

update doc pls,

**Reply by BonderTam** [reply]
finally got it.

![image](https://user-images.githubusercontent.com/249648/190080891-50f39b62-0782-4315-9d5d-ad98a1fc4071.png)
thx @simeng-li

**Reply by gao-sun** [reply]
@BonderTam thank you for reporting! the error will be fixed once this PR is merged: https://github.com/logto-io/docs/pull/264
